AppServer/google/appengine/tools/devappserver2/server.py

```diff
@@ -34,11 +35,12 @@
 import urlparse
 import wsgiref.headers
 
+
 from google.appengine.api import api_base_pb
 from google.appengine.api import apiproxy_stub_map
 from google.appengine.api import appinfo
 from google.appengine.api import request_info
-from google.appengine.api.logservice import log_service_pb
+from google.appengine.api.logservice import logging_capnp
```

Comment:
@cdonati the symlink didn't work well with our rsync_files. The build should copy that file over.